site/public/css/server.css

```diff
@@ -691,6 +691,10 @@ td > a.active-sort {
     margin-bottom: 10px;
 }
 
+code {
+    word-wrap: break-word;
+}
```

Comment:
This is not necessarily in the right spot.  As it is now, you're affecting all `<code>` elements everywhere on the system, which may be unintended.  You should find a more specific selector to target a specific element on this page.

Reply:
Alright, Thank You!